apps/web/sentry.server.config.ts

```diff
@@ -0,0 +1,6 @@
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
+  tracesSampleRate: 0,
```

Comment:
We will not be adding support for performance tracing right now.

Reply:
Worth this being an env var as well? In case we need to change in the future.

Reply:
I was thinking we'd add it in the future when we want to enable it. This was meant to intentionally disable it